src/domain/swaps/swaps.repository.e2e-spec.ts

```diff
@@ -0,0 +1,139 @@
+import { INestApplication } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import '@/__tests__/matchers/to-be-string-or-null';
+import { CacheKeyPrefix } from '@/datasources/cache/constants';
+import { SwapsModule } from '@/domain/swaps/swaps.module';
+import { ValidationModule } from '@/validation/validation.module';
+import { TestLoggingModule } from '@/logging/__tests__/test.logging.module';
+import { ConfigurationModule } from '@/config/configuration.module';
+import { NetworkModule } from '@/datasources/network/network.module';
+import { SwapsRepository } from '@/domain/swaps/swaps.repository';
+import { Order } from '@/domain/swaps/entities/order.entity';
+import configuration from '@/config/entities/configuration';
+
+const orderIds = {
```

Comment:
Nit: (for later) we should consider creating a builder for this, especially the stringified parts, e.g. `fullAppData`.

Reply:
Yes. Since this was an E2E test I wanted to specify the exact payload expected. At its current stage, for this test, we would have to specify almost every field of the builder.